From 60ff8660de508055b6926e45f6f30561baf61005 Mon Sep 17 00:00:00 2001
From: iurly <gerlando.falauto@gmail.com>
Date: Thu, 3 May 2018 10:41:37 +0200
Subject: [PATCH] node-red-pi: fix behavior with old bash version (#1713)

For some reason the following will result
in an endless loop under bash-4.3.42:

while([ -h "${SCRIPT_PATH}" ])

Just remove the round brackets (parentheses) to fix the issue.
They're not needed anyway.
---
 bin/node-red-pi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/node-red-pi b/bin/node-red-pi
index 828e5cca3..0e9de3c49 100755
--- a/bin/node-red-pi
+++ b/bin/node-red-pi
@@ -31,7 +31,7 @@ done
 # Find the real location of this script
 CURRENT_PATH=`pwd`
 SCRIPT_PATH="${BASH_SOURCE[0]}";
-while([ -h "${SCRIPT_PATH}" ]); do
+while [ -h "${SCRIPT_PATH}" ]; do
     cd "`dirname "${SCRIPT_PATH}"`"
     SCRIPT_PATH="$(readlink "`basename "${SCRIPT_PATH}"`")";
 done
